src/frontend/dune

```diff
@@ -21,6 +21,7 @@
   -open Merlin_analysis
   -open Merlin_kernel)
  (libraries
+  merlin_extend
```

Comment:
As far as I can tell, `merlin_extend` isn't used in `src/frontend` (except transitively via `merlin_kernel` and `merlin_analysis`). I'm basing this on a grep for `[Mm]erlin_extend` in the directory coming up empty.